pm_yaml_config/pm_yaml_config.xml

```diff
@@ -0,0 +1,126 @@
+                <param type="text" name="dftb_optionals" value="[]" optional="true" label="dftb optionals" help="additional optional json files to activate for DFTBArgs (for example, dftd3.json will use DFTD3 dispersion forces for 3ob-3-1 if DFTB+ has been compiled to support them)."/>
+                <param type="select" name="dftb_pbc" value="true" optional="true" label="dftp pbc" help="whether to turn on periodic boundary conditions in DFTB+.">
+                    <option value="true">true</option>
+                    <option value="false">false</option>
+                </param>
+            </when>
+        </conditional>
+        <param type="text" name="clustering_method" label="clustering method" value="hier" />
+        <param type="float" name="clustering_hier_t" value="0.2" label="hierarchical clustering t"/>
+        <param type="integer" name="supercell" value="30" label="supercell" multiple="true" help="supercell size and shape to use. This can either be a single int, a list of three integers or a 3x3 matrix of integers. For a single number a diagonal matrix will be generated with the integer repeated on the diagonals. For a list of three numbers a diagonal matrix will be generated where the digonal elements are set to the list. A matrix will be used directly as is. Default is a 3x3 indentity matrix."/>
+        <param type="text" name="k_points_grid" value="[1,1,1]" label="k-points grid" multiple="true" help="list of three integer k-points. Default is [1,1,1]."/>
+        <param type="integer" name="max_scc_steps" value="200" label="max scc steps" help="if applicable, max number of SCC steps to perform before giving up. Default is 200 which is also the default for DFTB+."/>
+    </inputs>
+    <outputs>
+        <data label="configuration" name="out_yaml" format="txt" from_work_dir="output.yaml"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="poisson_r" value="0.8"/>
+            <param name="struct_name" value="Si"/>
+            <param name="charged" value="true"/>
+            <param name="geom_steps" value="300"/>
+            <param name="vwd_scale" value="0.25"/>
+            <param name="out_folder" value="Si-out"/>
+            <param name="geom_force_tol" value="0.05"/>
+            <conditional name="calculator_cond">
+                <param name="calculator" value="uep"/>
+                <param name="uep_gw_factor" value="0.4"/>
+                <param name="uep_chden" value="Si.den_fmt"/>
+            </conditional>
+            <param name="clustering_method" value="hier"/>
+            <param name="clustering_hier_t" value="0.2"/>
+            <param name="supercell" value="30" />
+            <param name="k_points_grid" value="[1,1,1]"/>
+            <param name="max_scc_steps" value="200"/>
+            <output name="out_yaml" file="config.yaml" ftype="txt" compare="sim_size" delta_frac="0.001"/>
```

Comment:
Do we expect any difference at all here?

Reply:
there shouldnt be any

Reply:
Ok, then we can remove the `compare` and `delta_frac` options (it'll default to diff)

Reply:
apparently there are slight differences in metadata(history_data vs local_file) that cause it to fail on diff

Reply:
Wild. Ok, the change you've made is fine in that case.